Owncloud iOs Client/Network/Server Version Checks/CheckCapabilities.m

```diff
@@ -66,8 +68,9 @@ - (void) updateServerCapabilitiesOfActiveAccount {
                 app.activeUser.capabilitiesDto = [CapabilitiesDto new];
                 app.activeUser.capabilitiesDto = cap;
                 
-                //update file list view
-                [self reloadFileList];
+                //update file list view if needed
+                if(capabilitiesShareAPIChanged)
```

Comment:
Put the {} always even if it is just one line because it help to understand it :+1: